operations/fix-cache-headers.js

```diff
@@ -1,6 +1,8 @@
-module.exports = function fixCacheHeaders(headers) {
+var modifyHeaders = require('../modify-headers.js');
```

Comment:
Same thing here, I think it's confusing that this isn't the same as the `modifyHeaders` in `modify-headers.js`, and is instead the `module.exports` wrapper/factory around `modifyHeaders`